projects/angular/src/forms/styles/_properties.forms.scss

```
@@ -96,6 +96,26 @@
       // Range
       --clr-forms-range-progress-fill-color: var(--clr-color-action-600);
       --clr-forms-range-track-color: var(--clr-color-neutral-200);
+
+      // Password
+      --clr-forms-password-input-height: #{forms-variables.$clr-forms-password-input-height};
+      --clr-forms-password-padding-bottom: #{forms-variables.$clr-forms-password-padding-bottom};
+      --clr-forms-password-layout-space: #{forms-variables.$clr-forms-password-layout-space};
+      --clr-forms-password-eye-icon-min-height: #{forms-variables.$clr-forms-password-eye-icon-min-height};
+      --clr-forms-password-eye-icon-width: #{forms-variables.$clr-forms-password-eye-icon-width};
+
+      --clr-forms-password-default-typography-color: #{forms-variables.$clr-forms-password-default-typography-color};
+      --clr-forms-password-default-underline-color: #{forms-variables.$clr-forms-password-default-underline-color};
```

```suggestion
      --clr-forms-password-typography-color: #{forms-variables.$clr-forms-password-default-typography-color};
      --clr-forms-password-underline-color: #{forms-variables.$clr-forms-password-default-underline-color};
```
I don't think we need the `default` keyword as it's obvious and marking `disabled` only is enough.